packages/icons/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@ultraviolet/icons",
+  "version": "0.0.1",
+  "description": "Ultraviolet Icons",
+  "homepage": "https://github.com/scaleway/ultraviolet#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scaleway/ultraviolet",
+    "directory": "packages/icons"
+  },
+  "keywords": [
+    "react",
+    "reactjs",
+    "ui"
+  ],
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=18.x",
+    "pnpm": ">=8.0.0"
+  },
+  "os": [
+    "darwin",
+    "linux"
+  ],
+  "sideEffects": false,
+  "main": "dist/src/index.js",
+  "module": "dist/src/index.js",
+  "jsnext:main": "dist/src/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
Can we use directly the new export format from @QuiiBz PR ?

Reply:
https://github.com/scaleway/scaleway-lib/pull/1380